soroban-env-host/src/native_contract/token/storage_types.rs

```diff
@@ -2,6 +2,9 @@ use crate::native_contract::base_types::Address;
 use soroban_env_common::TryIntoVal;
 use soroban_native_sdk_macros::contracttype;
 
+pub(crate) const INSTANCE_BUMP_AMOUNT: u32 = 34560; // 2 hours
```

Comment:
Maybe this should be 1 day (24h)? So that the token that someone uses at least once per day is self-sustainable. SAC instance bumps are also very cheap as the instance itself is tiny. So maybe even 1 week wouldn't be offensive.

Reply:
My comment is actually incorrect. This is actually 2 days, not 2 hours.

Reply:
Cool, then please update the comment and then it seems good enough.